docs/UserGuide.adoc

```diff
@@ -172,35 +172,83 @@ Lists all the commands that you have entered in reverse chronological order. +
 Format: `history` +
 Alias: `hist`
 
-=== Creating a new journal entry : `new`
+[NOTE]
+====
+Pressing the kbd:[&uarr;] and kbd:[&darr;] arrows will display the previous and next input respectively in the command box.
+====
+
+=== Viewing a timetable : `tview`
+
+Shows the current saved timetable. +
+Format: `tview` +
+Alias: `tv`
+
+=== Adding a timetable : `tadd`
+
+Imports a timetable from NUSMODS website. +
+Format: `tadd` +
+Alias: `ta`
+
+=== Replacing a timetable : `treplace`
+
+Replaces the current timetable with an updated version. +
+Format: `treplace` +
+Alias: `tr`
+
+=== Viewing all journal entries : `jview`
+
+View all journal entries. +
+Format: `jview` +
+Alias: `jv`
+
+=== Creating a new journal entry : `jnew`
+
+Creates a new window that allows the user to input and save journal entries. +
+Format: `jnew` +
+Alias: `jn`
+
```

Comment:
Maybe if don't want to add a date field for this command, can make it such that the journal can only be added on today's date.

Format:  `jnew [d/DATE]`
Alias:  `jn [d/DATE]`

Example:
- `jnew 16-03-2018`
- `jn 16-03-2018`

Reply:
Yeah i was thinking of that too! a command which lets you assign a journal entry to a past or future doesn't make sense since a journal is a records of your current thoughts and feelings. I am not entirely sure how to implement that unless i make it so that the jnew command gets the current date and uses that to name the new entry automatically. But this means the user cannot name any entry, so it may be hard to find specific entries.

Reply:
Java has a function to call out current date